.travis.yml

```diff
@@ -1,9 +1,12 @@
-sudo: false
+sudo: required
 language: elixir
 elixir:
   - 1.2.0
-services:
-  - elasticsearch
+
+before_install:
+  - curl -O https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.1.1/elasticsearch-2.1.1.deb && sudo dpkg -i --force-confnew elasticsearch-2.1.1.deb
+  - sudo service elasticsearch start
```

Comment:
After starting should we:
`wget -nv --retry-connrefused --no-check-certificate -T 60 localhost:9200`
?
to wait for a proper elasticsearch restart?